.github/workflows/lintbuildandtest.yml

```diff
@@ -33,7 +33,7 @@ jobs:
         run: npm --prefix functions run build
       - name: Test
         id: test
-        run: npm --prefix functions run test
+        run: npm --prefix functions run test:coverage
```

Comment:
Would be great if we can upload the coverage results to CodeCov as we do for the other projects. 

I added a `CODECOV_TOKEN` to the set of secrets in this repo.

Reply:
on it - thanks!

Reply:
Thank you!

Reply:
Since the CodeCov setup is unfortunately not working, I would merge it as-is to still have code-coverage checks in the tests even without using the CodeCov tool. I will create a ticket to move the tests to use Typescript directly, put a low priority on it and include that it should include CodeCov then, okay?